frontend/src/app/pages/status/status.component.ts

```diff
@@ -36,7 +36,8 @@ export class StatusPage implements OnInit {
   constructor(
     private OrderService: OrderService,
     private GenericFuncs: GenericFunctionsService,
-    private router: Router
+    private router: Router,
+    private dataService: SendPageDataService
```

Comment:
your service vars are not consistent - mix of upper a lower case.

also for private vars I prefer the _ notations 


e.g
 private OrderService: OrderService,

would be
 private _orderService: OrderService,

Reply:
_ notation seems nice. I'll switch to it here. Won't be overnight but, I'll eventually propagate this notation style around the rest of the front end as well.

Reply:
Yea thats totally cool!

Reply:
roger that